cmd/activator/main.go

```diff
@@ -138,7 +138,7 @@ func main() {
 		log.Fatal("Error loading/parsing logging configuration:", err)
 	}
 	logger, atomicLevel := pkglogging.NewLoggerFromConfig(loggingConfig, component)
-	logger = logger.With(zap.String(logkey.ControllerType, "activator"))
+	logger = logger.With(zap.String(logkey.ControllerType, networking.ActivatorServiceName))
```

Comment:
I think it should be `component` name https://github.com/knative/serving/blob/master/cmd/activator/main.go#L70

Reply:
I am thinking because zap.String constructs field with key and value and here key is `logkey.ControllerType` which is `"knative.dev/controller"` so that value should be the controller name 
```
kubectl get deployment -n knative-serving
NAME                        READY   UP-TO-DATE   AVAILABLE   AGE
activator                   1/1     1            1           64s
autoscaler                  1/1     1            1           63s
controller                  1/1     1            1           58s
```